.npmignore

```diff
@@ -0,0 +1,14 @@
+.git/

```

Comment:
I don't like using `.npmignore` files. I'd much prefer using `files` property in `package.json`  to be more explicit about what is included in the package. 

https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d

Reply:
I'll make this change when I'm home